src/document/crdt/text.ts

```diff
@@ -77,6 +77,13 @@ export class CRDTTextValue {
     this.attributes.set(key, value, updatedAt);
   }
 
+  /**
+   * `getAttr` returns the attributes of this value.
+   */
+  public getAttr(): RHT {
```

Comment:
It seems that RHT should not be exposed.

Reply:
By the way, the name of this function is I think `getAttrs` would be better.